mongodb/src/main/scala/akka/stream/alpakka/mongodb/ObservableToPublisher.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package akka.stream.alpakka.mongodb
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import org.mongodb.{scala => mongoDB}
+import org.{reactivestreams => rxStreams}
+
+import scala.language.implicitConversions
+
+case class ObservableToPublisher[T](observable: mongoDB.Observable[T]) extends rxStreams.Publisher[T] {
```

Comment:
This looks like a simple `Publisher` but I'm still not 100% sure that it's completely correct. Normally we recommend such things to be implemented as `GraphStage`. Perhaps it's worth implementing it as `Publisher` because it's almost a one-to-one mapping to `mongoDB.Observable`?